UMLEditor/UMLEditor/Views/AStarSharp.cs

```diff
@@ -0,0 +1,218 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+
+namespace AStarSharp
+{
+    /// <summary>
+    /// Class to represent a node in the grid
+    /// </summary>
+    public class Node
+    {
+        // Change this depending on what the desired size is for each element in the grid
+        public static int NODE_SIZE = 15;
```

Comment:
NODE_Size should be commented like any other public member to address the warnings.